### DISCUSSION TITLE
I would like to upload a very large project to github

### DISCUSSION THREAD

**Discussion by LucasAstley** [community/community#65849] [New to GitHub]
Hello everyone, I recently created a big vanilla Minecraft map with custom generation using the Terralith, incendium and nullscape datapack. This map is pre-generated and the chunks are pre-loaded with a size of 10000x10000 blocks. The project is therefore around 50GB, I wanted to know if Github accepts such a size? Should we request special access?

**Comment by seyLu** [comment]
Keep in mind that Git's primary purpose is version control, not large-scale file storage. While GitHub offers Git LFS for managing large files, it's important to note that there are still limits, such as 5 GB for GitHub Enterprise Cloud and 2 GB for free users. For storing 50 GB of data, you might want to consider alternative cloud storage solutions.
